src/utils/injectControls.js

```diff
@@ -318,11 +322,10 @@ function createContextMenu() {
     }
 }
 
-function createTopMiddleContent() {
+function createHistoryButtons() {
     try {
-        const center_content = document.getElementsByTagName(
-            'ytmusic-pivot-bar-renderer'
-        )[0]
+        const search_box = document.getElementsByClassName('search-box')[0]
```

Comment:
![image](https://github.com/ytmdesktop/ytmdesktop/assets/17199186/ec9063dd-e1f7-44bb-a762-e35affd0c556)
Feel like it should go outside of the Search box to the left as shown if anything.

Reply:
Yeah, it's a lot harder change to make that happen due to the way the html is structured.  I played with it for a little.  I'll play with it again a little more later but it's not nearly as simple.

Reply:
Figured it out, will be in the next commit